hydra-node/src/Hydra/Chain/Direct/State.hs

```diff
@@ -918,3 +920,13 @@ unsafeObserveInitAndCommits ctx txInit commits =
       pure $ case event of
         OnCommitTx{committed} -> committed
         _ -> mempty
+
+chainSlotFromPoint :: ChainPoint -> ChainSlot
+chainSlotFromPoint p =
+  let (SlotNo s) = slotNoFromPoint p
+   in ChainSlot $ fromIntegral s
+
+slotNoFromPoint :: ChainPoint -> SlotNo
+slotNoFromPoint = \case
+  ChainPointAtGenesis -> 0
+  ChainPoint s _ -> s
```

Comment:
I saw this code before. From `Direct.Handlers`? If yes, let's DRY it.

Should: DRY this by moving `chainSlotFromPoint` next to the definition of `ChainSlot`

Reply:
Okay, it's already DRY. @ffakenz I will just move it further up in the module. This is the `danger zone` according to the haddocks ;)

Reply:
And it seems there is `chainPointToSlotNo` from `cardano-api` which we can use.